src/moai-android-adcolony/host.cpp

```diff
@@ -1,6 +1,16 @@
 // Copyright (c) 2010-2011 Zipline Games, Inc. All Rights Reserved.
 // http://getmoai.com
 
+// From moai-box2d's pch.h
+#include <moai_config.h>
+#include <zl-util/headers.h>
+#include <moai-core/headers.h>
+#include <moai-sim/headers.h>
```

Comment:
Well, just checked MOAIAdColonyAndroid.cpp and it is already completely wrappered by #if AKU_WITH_ANDROID_ADCOLONY, but it also has not pulled in <host-modules/aku_modules_android_config.h> prior to referencing AKU_WITH_ANDROID_ADCOLONY.  The configurability of the Android host plug-ins isn't quite up to snuff, IMHO.